x/programs/examples/token.go

```diff
@@ -28,6 +28,11 @@ func NewToken(log logging.Logger, programBytes []byte, db state.Mutable, cfg *ru
 	}
 }
 
+type minter struct {
+	To     [32]byte
```

Comment:
Node that Address is 33 bytes. I recommend using `Address` directly here, if possible: https://github.com/ava-labs/hypersdk/blob/283846bac51de111d18c225c8b0a7b92b6cf6ec8/codec/address.go

Reply:
will create an issue to move to `HyperSDK.address` later. Currently use ed25519 public keys instead of addresses which have 32 bytes and not 33.